From f5e95695f5e26447a73babf06e6226ef83b7aa15 Mon Sep 17 00:00:00 2001
From: Miao Yu <r65093@freescale.com>
Date: Tue, 8 Dec 2009 16:12:43 +0800
Subject: [PATCH 138/505] ENGR00119073 TVIN: Remove the crop setting when tvin
 preview is started

The crop setting only work fine before the preview is started. For
the current crop setting is overridden by VIDIOC_S_PARM issue, it
is resolved in bsp layer(engr119069).

Signed-off-by: Miao Yu <r65093@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_tvin.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_tvin.c b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
index 8ebac5e..834155e 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_tvin.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
@@ -72,7 +72,6 @@ mxc_v4l_tvin_test(void)
         int overlay = 1;
 	v4l2_std_id id;
 	struct v4l2_streamparm parm;
-	struct v4l2_crop crop;
 
         parm.type = V4L2_BUF_TYPE_VIDEO_CAPTURE;
         parm.parm.capture.timeperframe.numerator = 1;
@@ -104,16 +103,6 @@ mxc_v4l_tvin_test(void)
 		    g_cap_mode = (g_cap_mode + 1) % 2;
 		    parm.parm.capture.capturemode = g_cap_mode;
 		    ioctl(fd_v4l, VIDIOC_S_PARM, &parm);
-		    crop.type = V4L2_BUF_TYPE_VIDEO_OVERLAY;
-		    crop.c.left = 0;
-		    crop.c.top = 0;
-		    crop.c.width = PAL_WIDTH;
-		    crop.c.height = PAL_HEIGHT;
-		    if (ioctl(fd_v4l, VIDIOC_S_CROP, &crop) < 0)
-        	    {
-			printf("set cropping failed\n");
-			return TFAIL;
-		    }
 		    g_current_std = id;
                     sleep(1);
 		    continue;
-- 
1.8.0

